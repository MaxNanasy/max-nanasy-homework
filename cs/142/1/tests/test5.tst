main(){type myinteger=short;var foo,bar:long;const FALSE=0;const TRUE=1;foo=1;bar=2;while(foo!=10){foo=foo+1;bar=(bar*foo)/2;}if(bar>100){print 1,3,3,7;}}