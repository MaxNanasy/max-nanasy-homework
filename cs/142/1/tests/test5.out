 MAIN  (  )  {  TYPE  ident  =  SHORT  ;  VAR  ident  ,  ident  :  ident  ;  CONST  ident  =  number  ;  CONST  ident  =  number  ;  ident  =  number  ;  ident  =  number  ;  WHILE  (  ident  !=  number  )  {  ident  =  ident  +  number  ;  ident  =  (  ident  *  ident  )  /  number  ;  }  IF  (  ident  >  number  )  {  PRINT  number  ,  number  ,  number  ,  number  ;  }  }  <EOT>
